_mental_models/saying-no.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: saying-no
+name: Saying "No"
+benefit: Better Decisions
```

Comment:
Benefit is actually something like "protecting decisions" or "removes outside interference"

though better decision is also there.. so I'll let you decide which you think will happen